midp/gfx.js

```diff
@@ -59,6 +59,7 @@ var currentlyFocusedTextEditor;
 
     Native.create("com/sun/midp/lcdui/DisplayDevice.gainedForeground0.(II)V", function(hardwareId, displayId) {
         console.warn("DisplayDevice.gainedForeground0.(II)V not implemented (" + hardwareId + ", " + displayId + ")");
+        document.getElementById("splash-screen").style.display = "none";
```

Comment:
I believe we need to modify this function to fix #823, but that doesn't necessarily mean that this warning is useful to us